base/audit_events.go

```diff
@@ -18,13 +18,105 @@ const (
+	//AuditIDBLIPMessage           AuditID = 0 // TODO: Is this required? What info should the event contain?
+
+	// Auth (public) events
+	AuditIDPublicUserAuthenticated        AuditID = 53280
+	AuditIDPublicUserAuthenticationFailed AuditID = 53281
+	AuditIDPublicUserSessionCreated       AuditID = 53282
+	AuditIDPublicUserSessionDeleted       AuditID = 53283
+	// Auth (admin) events
+	AuditIDAdminUserAuthenticated        AuditID = 53290
+	AuditIDAdminUserAuthenticationFailed AuditID = 53291
+	AuditIDAdminUserAuthorizationFailed  AuditID = 53292
+
+	// SG node events
+	AuditIDSyncGatewayCollectInfoStatus AuditID = 53300
+	AuditIDSyncGatewayCollectInfoStart  AuditID = 53301
+	AuditIDSyncGatewayCollectInfoStop   AuditID = 53302
+	AuditIDSyncGatewayStats             AuditID = 53303
+	AuditIDSyncGatewayProfiling         AuditID = 53304
+
+	// SG cluster events
+	AuditIDClusterInfoRead AuditID = 53350
+
+	// Database events
+	AuditIDCreateDatabase  AuditID = 54000
+	AuditIDReadDatabase    AuditID = 54001
+	AuditIDDeleteDatabase  AuditID = 54002
+	AuditIDDatabaseAllRead AuditID = 54003
+	// Database config events
+	AuditIDReadDatabaseConfig   AuditID = 54010
+	AuditIDUpdateDatabaseConfig AuditID = 54011
+	// Database operation events
+	AuditIDDatabaseOffline       AuditID = 54020
+	AuditIDDatabaseOnline        AuditID = 54021
+	AuditIDDatabaseCompactStatus AuditID = 54030
+	AuditIDDatabaseCompactStart  AuditID = 54031
+	AuditIDDatabaseCompactStop   AuditID = 54032
+	AuditIDDatabaseResyncStatus  AuditID = 54040
+	AuditIDDatabaseResyncStart   AuditID = 54041
+	AuditIDDatabaseResyncStop    AuditID = 54042
+	AuditIDDatabasePostUpgrade   AuditID = 54043
+	AuditIDDatabaseRepair        AuditID = 54044
+	AuditIDDatabaseFlush         AuditID = 54045
+
+	// User principal events
+	AuditIDUserCreate AuditID = 54100
+	AuditIDUserRead   AuditID = 54101
+	AuditIDUserUpdate AuditID = 54102
+	AuditIDUserDelete AuditID = 54103
+	// Role principal events
+	AuditIDRoleCreate AuditID = 54110
+	AuditIDRoleRead   AuditID = 54111
+	AuditIDRoleUpdate AuditID = 54112
+	AuditIDRoleDelete AuditID = 54113
+
+	// Changes feeds events
+	AuditIDChangesFeedStarted AuditID = 54200
+
+	// BLIP Replication events
+	AuditIDReplicationConnect    AuditID = 54300
+	AuditIDReplicationDisconnect AuditID = 54301
+
+	// ISGR events
+	AuditIDISGRCreate    AuditID = 54400
+	AuditIDISGRRead      AuditID = 54401
+	AuditIDISGRUpdate    AuditID = 54402
+	AuditIDISGRDelete    AuditID = 54403
+	AuditIDISGRStatus    AuditID = 54410
+	AuditIDISGRStart     AuditID = 54411
+	AuditIDISGRStop      AuditID = 54412
+	AuditIDISGRReset     AuditID = 54413
+	AuditIDISGRAllStatus AuditID = 54420
+	AuditIDISGRAllRead   AuditID = 54421
+
+	// Documents events
+	AuditIDDocumentCreate       AuditID = 55000
+	AuditIDDocumentRead         AuditID = 55001
+	AuditIDDocumentUpdate       AuditID = 55002
+	AuditIDDocumentMetadataRead AuditID = 55004
+	// Document attachments events
+	AuditIDDocumentDelete   AuditID = 55003
```

Comment:
switch order for numeric consistency and comment association

```suggestion
	AuditIDDocumentDelete   AuditID = 55003
	AuditIDDocumentMetadataRead AuditID = 55004
	
	// Document attachments events
```